src/Cli/func/Helpers/PythonHelpers.cs

```diff
@@ -500,8 +500,20 @@ private static async Task RestorePythonRequirementsDocker(string functionAppRoot
                 dockerImage = await ChoosePythonBuildEnvImage();
```

Comment:
I feel like this (ChoosePythonBuildEnvImage) should handle returning the local docker file name.

`public const string LinuxPython313ImageAmd64 = "mcr.microsoft.com/azure-functions/python:4-python3.13-buildenv";`

Can be replaced with something like `local/python3.13-buildenv`

`ChoosePythonBuildEnvImage` can be updated to return a tuple: imageName, isLocal

Then here below on line 503, we can check ifLocal = true and handle the image that way - this makes it more generic and not just for 313

Reply:
This is just a quick thought - feel free to clean this up for something that makes more sense but the change should ensure we make these changes easy to expand on in future